src/css/components/confirm.scss

```diff
@@ -18,10 +18,11 @@
 
 .confirm-over {
   align-items: center;
+  background-color: #f5f5f3;
```

Comment:
We shouldn't be using straight color values anywhere, everythign should come from the variables defined in `override_vars`.  So this should be `background-color: $color-lightestgray;`